dev/io.openliberty.microprofile.reactive.messaging30.internal_fat_tck/fat/src/io/openliberty/microprofile/reactive/messaging30/tck/ReactiveMessagingTCKLauncher.java

```diff
@@ -53,7 +54,8 @@ public static void tearDown() throws Exception {
 
     @Test
     @Mode(TestMode.FULL)
-    @AllowedFFDC({ "org.jboss.weld.exceptions.DeploymentException", "com.ibm.ws.container.service.state.StateChangeException", "jakarta.enterprise.inject.spi.DeploymentException" }) // The tested deployment exceptions cause FFDC so we have to allow for this.
```

Comment:
Given this is the RM 30 TCK, this will throw `jakarta.*` exceptions, not javax, so this wouldn't stop a failure in the case of the issue